### PR TITLE
fix: Fixes PIL resizing for non squared shapes

### PR DESCRIPTION
**Pull request by frgfm**
This PR introduces the following modifications:
- fixes the resizing for non-squared images
- updates unittests expected shapes
- updates unittests model to save bandwidth

Any feedback is welcome!

### CODE REVIEW

File: pyroengine/vision.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ def preprocess_image(self, pil_img: Image.Image) -> np.ndarray:
         """
 
         # Resizing
-        img = pil_img.resize(self.cfg["input_shape"][-2:], Image.BILINEAR)
+        img = pil_img.resize(self.cfg["input_shape"][-2:][::-1], Image.BILINEAR)
```
Comment (MateoLostanlen) [inline pyroengine/vision.py:50]: Please use Image.Resampling.BILINEAR instead, Image.BILINEAR is [deprecated](https://pillow.readthedocs.io/en/stable/deprecations.html)

**Review by MateoLostanlen — CHANGES_REQUESTED** [review]
Thanks for this PR, just on small change

File: pyroengine/vision.py (hunk 1 above)
Comment (frgfm) [reply pyroengine/vision.py:50]: This change would require also to update the version specifiers for Pillow, so I suggest doing it in another PR :+1: 

(either putting <9.1.0 in the specifier, or going for your change + >=X.Y.Z, where X.Y.Z is the first release where Resampling become supported)

Comment (MateoLostanlen) [reply pyroengine/vision.py:50]: OK let's stay like this then

**Review by MateoLostanlen — APPROVED** [review]

**Pull request merged**